### PR TITLE
[11.0][ADD] stock_picking_auto_create_lot: Create lot for incoming pickings

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: stock_picking_auto_create_lot/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    def button_validate(self):
+        if self.picking_type_code == 'incoming':
```
Comment (rousseldenis) [inline stock_picking_auto_create_lot/models/stock_picking.py]: @sergio-teruel IMHO to be flexible, this should be an option in stock.picking.type (doing this you limit lot auto creation to incoming pickings).

So,

```suggestion
        if self.picking_type_id.auto_create_lot:
```

File: stock_picking_auto_create_lot/tests/test_stock_picking_auto_create_lot.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import SavepointCase
+
+
+class TestStockPickingAutoCreateLot(SavepointCase):
+    at_install = False
+    post_install = True
```
Comment (rousseldenis) [inline stock_picking_auto_create_lot/tests/test_stock_picking_auto_create_lot.py]: I don't think this module breaks normal behaviour

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Minor comment

**Comment by sergio-teruel** [comment]
@rousseldenis Changes done!!. About add field to operation types, I was a bit lazy... :smiley:

**Review by rousseldenis — APPROVED** [review]
LGTM. Code review

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**